sds_data_manager/lambda_code/SDSCode/create_schema.py

```diff
@@ -5,9 +5,10 @@
 import sys
 
 import requests
-from SDSCode.database.database import engine
 from SDSCode.database.models import Base
 
+from .database import database as db
```

Comment:
I also am not a huge fan of `from datetime import datetime` like imports, so in a follow-up PR maybe we could move this to `initialization.py` or something like that? Or even move all the models and initialization code into a single file if it isn't too much code.